misk-schema-migrator-gradle-plugin/src/test/resources/schema-migrator-plugin-test/gradle.properties

```diff
@@ -0,0 +1 @@
+org.gradle.configuration-cache=true
```

Comment:
Not sure if this is needed, I think it's default in new Gradle versions.

Reply:
It's still off by default in 8.8 https://docs.gradle.org/current/userguide/configuration_cache.html

I want to make sure that we don't accidentally regress and break config cache support

Reply:
It is proposed to be default in Gradle 9. I think it makes sense to keep this for the reason Yissachar mentioned.